src/logger.ts

```diff
@@ -52,16 +52,16 @@ export class Logger {
   ) {}
 
   debug = (message: string, args: any = {}) => {
```

Comment:
Do you think it'd make sense to make this signature `(message: string, args: { [key: string]: value } = {})` so people using TypeScript will get an error if they send anything else?

Reply:
makes sense, updated.

Reply:
now the new test is broken because I can't pass a string 😂

Reply:
@arne fixed the test